src/framework/theme/components/calendar/_calendar.component.theme.scss

```diff
@@ -8,94 +8,60 @@
 
   nb-base-calendar {
     nb-card {
-      overflow: hidden;
-    }
-
-    &.medium nb-card {
-      width: nb-theme(calendar-width);
-
-      nb-card-body {
-        height: nb-theme(calendar-body-height);
-      }
-    }
-
-    &.large nb-card {
-      width: nb-theme(calendar-large-width);
-
-      nb-card-body {
-        height: nb-theme(calendar-large-body-height);
-      }
-    }
-
-    &.has-week-number.medium nb-card {
-      width: calc(#{nb-theme(calendar-width)} + #{nb-theme(calendar-day-cell-width)} + 1px);
-    }
-
-    &.has-week-number.large nb-card {
-      width: calc(#{nb-theme(calendar-large-width)} + #{nb-theme(calendar-day-cell-width)} + 1px);
-    }
-
-    nb-card {
-      border: none;
+      background-color: nb-theme(calendar-background-color);
+      border: nb-theme(calendar-border-width) nb-theme(calendar-border-style) nb-theme(calendar-border-color);
+      box-shadow: none;
       margin: 0;
+      width: nb-theme(calendar-width);
+      overflow: hidden;
     }
 
     nb-card-body {
       padding: 0;
-
-      & > nb-calendar-pageable-navigation, & > nb-calendar-navigation {
-        padding: 1rem 0;
-      }
     }
-  }
 
-  nb-calendar-day-picker .range-cell {
-    flex: 1;
-    margin: 1px 0;
-
-    .day-cell {
-      margin: 0;
+    .calendar-navigation {
+      border: none;
+      display: flex;
+
+      padding: nb-theme(calendar-navigation-padding);
+      color: nb-theme(calendar-navigation-text-color);
+      font-family: nb-theme(calendar-navigation-text-font-family);
+      font-size: nb-theme(calendar-navigation-title-text-font-size);
+      font-weight: nb-theme(calendar-navigation-title-text-font-weight);
+      line-height: nb-theme(calendar-navigation-title-text-line-height);
     }
 
-    &.in-range {
-      background: nb-theme(calendar-cell-in-range-background-color);
-
-      &:not(.today, :hover) {
-        border-radius: 0;
+    &:not(.has-navigation) {
+      nb-calendar-week-numbers,
+      nb-calendar-days-names {
+        border-top: 0;
```

Comment:
Don't you think it ought to be configured thought the theme?

Reply:
`nb-calendar-week-numbers` and `nb-calendar-days-names` borders defined in theme. This line removes ensures it's not shown when there is no navigation bar before. Otherwise, you'll have a double border like this:
![image](https://user-images.githubusercontent.com/6151971/70429529-ccfa0300-1a89-11ea-9622-217decaef7e4.png)